pkg/point/client.go

```diff
@@ -0,0 +1,118 @@
+package point
+
+import (
+	"context"
+
+	"github.com/mercadopago/sdk-go/pkg/config"
+	"github.com/mercadopago/sdk-go/pkg/internal/baseclient"
+)
+
+const (
+	urlBase                = "https://api.mercadopago.com"
+	paymentIntentUrl       = urlBase + "/point/integration-api/devices/:device_id/payment-intents"
+	paymentIntentSearchUrl = urlBase + "/point/integration-api/payment-intents/:payment_intent_id"
+	paymentIntentCancelUrl = urlBase + "/point/integration-api/devices/:device_id/payment-intents/:payment_intent_id"
+	devicesUrl             = urlBase + "/point/integration-api/devices"
+	deviceWithIDUrl        = urlBase + "/point/integration-api/devices/:device_id"
+)
+
+// client is the implementation of Client.
+type client struct {
+	cfg *config.Config
+}
+
+// Client contains the methods to interact with the Point API.
+type Client interface {
+
+	// Create a point payment intent.
+	// It is a post request to the endpoint: https://api.mercadopago.com/point/integration-api/devices/{device_id}/payment-intents
+	// Reference: https://www.mercadopago.com.br/developers/pt/reference/integrations_api_paymentintent_mlb/_point_integration-api_devices_deviceid_payment-intents/post
+	Create(ctx context.Context, deviceID string, request Request) (*CreateResponse, error)
+
+	// Search a point payment intent.
+	// It is a search request to the endpoint: https://api.mercadopago.com/point/integration-api/payment-intents/{payment_intent_id}
+	// Reference: https://www.mercadopago.com.br/developers/pt/reference/integrations_api/_point_integration-api_payment-intents_paymentintentid/get
+	Search(ctx context.Context, paymentIntentID string) (*SearchResponse, error)
```

Comment:
não deveria ser um Get, search entendo que seja quando utilizamos queryParam para buscar, igual as outras APIs
```suggestion
	Get(ctx context.Context, paymentIntentID string) (*SearchResponse, error)
```

Reply:
Alterado!!